pyramid_hypernova/request.py

```diff
@@ -74,7 +74,15 @@ def json(self):
         else:
             try:
                 result = self.response.wait()
-                json = result.json()
             except NetworkError as e:
                 raise HypernovaQueryError(e)
+            else:
+                # NetworkError is only called raised there's an actual network
+                # problem (socket closed, etc.) and not for non-2xx statuses.
+                if result.code != 200:
+                    raise HypernovaQueryError(
```

Comment:
Could we add the response body to this error? I could see that being useful information, especially when it's something other than a timeout.

Reply:
done